data/access/stories.yml

```diff
@@ -15,7 +15,8 @@ stories:
     steps:
       - action: utter_access_request
       - intent: intent_core_no
-      - checkpoint: check_closing
```

Comment:
What was happening with the checkpoint? o.O

Reply:
short story: I  have no idea
long story: I was looking and i have no idea - but there was something going on!

Reply:
Somethings enough of an idea for me :laughing: